### ISSUE TITLE
Crash when rescannning is finished

### ISSUE THREAD

**Issue by freebeego** [zcash/zcash#6111]
Nothing in the debug.log.

syslog:
```
kernel: [27894.952094] zcashd[4970]: segfault at 18 ip 000055f7b33dd5b6 sp 00007ffc209724c0 error 4 in zcashd[55f7b33d1000+d09000]
```

**Comment by sandakersmann** [comment]
More on the issue here:

https://forum.zcashcommunity.com/t/v5-2-0-segmentation-fault/42775

**Comment by freebeego** [comment]
Hi @str4d. Could some information from apport crash log helps you?

**Comment by daira** [comment]
Yes that would help. Also, which version of zcashd were you running? And can you reproduce the segfault reliably?

**Comment by daira** [comment]
If you can reproduce it, then one option would be to have a video call (e.g. Google Meet) and debug it that way.

**Comment by freebeego** [comment]
@daira, I texted you on the Discord. Tech support channel.
Built based on the last master branch version.
Yes I can.
